src/Sfa.Tl.ResultsAndCertification.Web/ViewModel/AdminDashboard/Assessment/AdminCoreComponentViewModel.cs

```diff
@@ -11,7 +11,7 @@ public class AdminCoreComponentViewModel : AdminAssessmentLearnerDetails
     {
         public AdminAssessmentDetailsViewModel AssessmentDetails { get; set; }
 
-        public IEnumerable<PathwayAssessments> ValidPathwayAssessmentSeries { get; set; }
+        public IList<PathwayAssessments> ValidPathwayAssessmentSeries { get; set; }
```

Comment:
Why this change? No method of IList is used anywhere

Reply:
Updated